package/MDAnalysis/analysis/encore/similarity.py

```diff
@@ -1283,7 +1273,7 @@ def dres(ensembles,
         classes from the dimensionality_reduction module. Different parameters
         for the same method can be explored by adding different instances of
         the same dimensionality reduction class. Provided methods are the
-        Stochastic Proximity Embedding (default) and the Principel Component
+        Stochastic Proximity Embedding (default) and the Principal Component
```

Comment:
So then you need to load everything into memory while the other implementation only needs to hold the current frame and can therefore estimate a PCA for even a TB trajectory.